pages/pil/unscoped/courses/participants/add/content/index.js

```diff
@@ -63,13 +63,28 @@ module.exports = {
     trainingNeed: {
       required: 'This is a required field.'
     },
+    organisation: {
+      required: 'Enter their organisation.'
+    },
+    qualificationLevelAndSubject: {
+      required: 'Enter their qualification level and subject.'
+    },
     applicantTrainingUseAtWork: {
       lessThanOrEqualToMaxWordCount:
-        'How the applicant will use this training must be 100 characters or less'
+        'How the applicant will use this training must be 100 characters or less',
+      required: 'Tell us how they will use this training in their work.'
```

Comment:
https://design-system.service.gov.uk/components/character-count/#if-the-input-is-empty
```suggestion
      required: 'Enter how the participant will use this training in their work.'
```